src/components/TextInput/TextInput.tsx

```diff
@@ -42,6 +50,7 @@ export const TextInput: React.FC<TextInputProps> = ({
   startAddon,
   startIcon,
   value,
+  type,
```

Comment:
this _probably_ needs to default to "text"
```suggestion
  type = "text",
```